test/classes/initializers/phase1/multipleInits2.good

```diff
@@ -0,0 +1,2 @@
+multipleInits2.chpl:10: error: multiple initializations of field "f2" detected
+multipleInits2.chpl:8:  note: previous initialization(s) were here
```

Comment:
alternatively, it would be reasonable for this error to be "fields not initialized in order"